appeals/web/src/server/appeals/appeal-details/__tests__/__snapshots__/appeal-details.test.js.snap

```diff
@@ -360,7 +360,7 @@ exports[`appeal-details GET /:appealId should render a Decision inset panel when
                         </div>
                         <div class="govuk-summary-list__row"><dt class="govuk-summary-list__key"> Start date</dt>
                             <dd class="govuk-summary-list__value">23 May 2023</dd>
-                            <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/start-case/date"> Change<span class="govuk-visually-hidden"> The date the case was started</span></a>
+                            <dd class="govuk-summary-list__actions"><a class="govuk-link" href="/appeals-service/appeal-details/2/#"> Change</a>
```

Comment:
We've gone from a complete url to a url with # in it?

Reply:
If this is tied to the fact that change is another ticket then we probably need a TODO and link the ticket to BOAT851